index.js

```diff
@@ -33,14 +41,14 @@ const blackListedSemesterRepos = [
 ];
 
 blackListedSeEduRepos.forEach(repo => {
-  BlackListed(accuser, 'se-edu', repo, 'practice-fork.mst');
+  Blacklisted(accuser, originAccount, repo, 'practice-fork.mst');
 });
 
 blackListedSemesterRepos.forEach(repo => {
-  BlackListed(accuser, semesterAccount, repo, 'practice-fork.mst');
+  Blacklisted(accuser, semesterAccount, repo, 'practice-fork.mst');
 });
 
-BlackListed(accuser, 'se-edu', 'rcs', 'practice-fork.mst');
+Blacklisted(accuser, originAccount, 'rcs', 'practice-fork.mst');
```

Comment:
Think we can remove this line, coz this repo is not used anymore. Also I think the original message template for this was `wrong-repository.mst`